### PR TITLE
feat: additional MOTD, grub entry, and os-release polish 

### PR DESCRIPTION
**Pull request by RoyalOughtness**
- removes Fedora trademark use, while adding a note that secureblue is "powered by Fedora Atomic"
- sets grub to `secureblue (powered by Fedora Atomic)`
- adds secureblue info to /usr/lib/os-release
- include glyphs in the MOTD to prettify it further
- include the nerdfont needed for glyphs in the motd, plus three other fonts that are unavailable from rpm

image

### CODE REVIEW

**Comment by fiftydinar** [comment]
You forgot to remove gdm logo:
`fedora-gdm-logo.png`

**Review by mintpilo — DISMISSED** [review]

**Comment by RoyalOughtness** [comment]
> You forgot to remove gdm logo: `fedora-gdm-logo.png`

overwritten in latest commit

**Comment by RoyalOughtness** [comment]
need to add `rpm-ostree override remove fedora-logos` instead

File: recipes/common/desktop-packages.yml
Comment (mintpilo) [inline recipes/common/desktop-packages.yml:1]: Is fedora-logos only in the desktop images, or would it also be bundled with the server images?

Comment (RoyalOughtness) [reply recipes/common/desktop-packages.yml:1]: just checked, coreos doesn't have them

**Review by mintpilo — APPROVED** [review]

**Review by RKNF404 — APPROVED** [review]

**Pull request merged**